control_center.yml

```diff
@@ -8,16 +8,15 @@
     - import_role:
         name: confluent.variables
 
+    - name: Populate service facts
+      service_facts:
+
     - name: Check if Control Center Service Running
-      shell: "systemctl show -p SubState {{control_center_service_name}}"
-      changed_when: false
-      # On some SELinux enabled hosts this command will error out, handled in next set_fact task
-      failed_when: false
-      check_mode: false
-      register: substate
+      set_fact:
```

Comment:
can probably remove this set_fact right? only really need the install_pattern var set

Reply:
I think it would become almost unreadable if doing so. What about a var on the set_fact task for install_pattern?

Reply:
oh that'd be interesting! that way we are not setting unused variables. And keeps readability

Reply:
@domenicbove PTAL